src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
```

Comment:
Why do you use setters here? There is a constructor in the class Ball to create an object with these 2 fields.
Also It's not required to create and variable you can simply ```return new Ball(1, 1);```